src/components/organisms/PaymentModal/PaymentConfirmation.tsx

```diff
@@ -1,22 +0,0 @@
-import React from "react";
-import { CountDown } from "components/molecules/CountDown";
```

Comment:
Is `components/molecules/CountDown` still used anywhere?

Reply:
Yes 
`VenueLandingPage` and `VenuePage`